db/seeds.rb

```diff
@@ -1,7 +1,32 @@
 # This file should contain all the record creation needed to seed the database with its default values.
 # The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
 #
-# Examples:
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
+# rails -T db           # => I can see every task for database
+# rails db:seed:replant # =>
+# The difference with db:reset you can execute with server running, because delete registre not database and run seeds
+# rails db:truncate_all # =>
+# The difference with db:destroy you can execute with server running because delete registre not database
+
+# WE WANT DENOUNCES A
```

Comment:
If all these comments are not necessary you can remove them, so that the code looks cleaner.